### ISSUE TITLE
Embed Video Via Web Editor?

### ISSUE THREAD

**Issue by ghost** [GitbookIO/plugin-youtube#19]
Hi. Me and my friends are using the web version of GitBook to build a book. We'd like to embed a YouTube video so that it can play inline within the context of the book. Is it possible to do this via the web editor, or do you have to actually download and build GitBook locally?  Thanks

**Comment by MikeUdin** [comment]
https://docs.gitbook.com/content-editing/embeds

**Comment by behu-kea** [comment]
I don't think this works. Not through github at least. I "fixed" it by making an screenshotting an image of the video and making an image with a link. https://github.com/HackYourFuture-CPH/teacher-curriculum/blob/master/README.md

Such a shame it does not work properly
